python/vote-text-bot-cloud-function/main.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START python-vote-text-bot]
```

Comment:
Convention for region tags is to omit language (it's implied from file extension) + prefer underscores. Use product prefix instead (e.g. chat_vote_text_bot)